torch/jit/_script.py

```diff
@@ -1221,6 +1221,14 @@ def _get_overloads(obj):
     if uncompiled_overloads is None:
         return existing_compiled_fns
 
+    if obj in uncompiled_overloads:
+        sourcelines, file_lineno, filename = _jit_internal.get_source_lines_and_file(obj)
+        raise RuntimeError(
```

Comment:
Same here, we might want to show complete usage guidance on `_overload` in this case.